manifests/vanilla/deploy-csi-snapshot-components.sh

```diff
@@ -156,13 +196,50 @@ check_and_deploy_crds(){
 	echo  -e "\n✅ Deployed VolumeSnapshot CRDs\n"
 }
 
+find_control_plane_metadata(){
+	declare -g node_tolerations=""
+	declare -g node_tolerations=""
```

Comment:
Not a shell script expert, but isn't `node_tolerations` already declared before?

Reply:
I think you're right, I've removed those.